app/src/androidTest/java/org/mozilla/focus/activity/TestHelper.java

```diff
@@ -312,4 +321,25 @@ static void waitForWebSiteTitleLoad() {
         assertTrue(webView.waitForExists(waitingTime));
         Assert.assertTrue("Website title loaded", titleMsg.waitForExists(waitingTime));
     }
+
+    // Checks whether keyboard is visible by looking at the coord of the logo w.r.t. container
+    public static boolean checkKeyboardPresenceInMainView() throws UiObjectNotFoundException {
+        UiObject container = TestHelper.mDevice.findObject(new UiSelector()
+                .resourceId("org.mozilla.focus.debug:id/container")
+                .enabled(true));
+        UiObject mainLabel = TestHelper.mDevice.findObject(new UiSelector()
+                .description("Firefox Focus")
```

Comment:
Let's start to use strings we get from the context object instead of plain English strings, for new code.

Reply:
will do